script/test_in_openshift/run.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Test case for kompose up/down with etherpad
+
+KOMPOSE_ROOT=$(readlink -f $(dirname "${BASH_SOURCE}")/../../..)
+#source $KOMPOSE_ROOT/script/test/cmd/globals.sh
+
+
+# Install oc client
+
+sudo apt-get install wget -y
+sudo sed -i 's:DOCKER_OPTS=":DOCKER_OPTS="--insecure-registry 172.30.0.0/16 :g' /etc/default/docker
+sudo mv /bin/findmnt /bin/findmnt.backup
+sudo cat /etc/default/docker
+sudo /etc/init.d/docker restart;
+wget https://github.com/openshift/origin/releases/download/v1.4.1/openshift-origin-client-tools-v1.4.1-3f9807a-linux-64bit.tar.gz -O /tmp/oc.tar.gz
+mkdir /tmp/ocdir && cd /tmp/ocdir && tar -xvvf /tmp/oc.tar.gz
+sudo mv /tmp/ocdir/*/oc /usr/bin/
+
+
+# Run oc cluster up
+oc cluster up; result=$?;
+
+if [ $result -ne 0 ]; then
+    echo "oc cluster up failed"
+    exit 1;
+fi
+
+oc login -u system:admin;
+
+# Env variables for etherpad
+export $(cat ${KOMPOSE_ROOT}/kompose/script/test/fixtures/etherpad/envs)
+
+# Run kompose up
+kompose --emptyvols --provider=openshift -f $KOMPOSE_ROOT/kompose/script/test/fixtures/etherpad/docker-compose.yml up; result=$?;
```

Comment:
It would be great to separate parts that are running tests from parts that are doing openshift setup.
Tests should be in different files from setup script